src/nav/apm.yml

```diff
@@ -1103,79 +1103,105 @@ pages:
       - title: Ruby monitoring
         path: /docs/apm/agents/ruby-agent
         pages:
-          - title: Monitor your Ruby app
+          - title: Introduction to Ruby monitoring
             path: /docs/apm/agents/ruby-agent/getting-started/introduction-new-relic-ruby
+          - title: Install the Ruby agent
+            path: /docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent
+          - title: Ruby configuration
+            path: /docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration
+          - title: Ruby custom instrumentation
+            path: /docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation
+          - title: Requirements and supported frameworks
+            path: /docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks
           - title: Compatibility and releases
```

Comment:
if compatibility is moving up to the top level, I think this category name needs to change